src/containers/MainScreen/index.js

```diff
@@ -45,8 +55,23 @@ class MainScreen extends Component {
     });
   }
 
+  renderMarkers = () => {
+    const { targets } = this.props;
+    const { zoom } = this.state;
+    return targets.map((obj) => {
```

Comment:
```suggestion
return targets.map(({ target: { id, lat, lng, radius, topicId } }) => {
```
you could desctructure the object